### PR TITLE
docs: add extra permissions for EKS addon installation

### PR DESCRIPTION
**Pull request by salasberryfin**
Add extra permissions for EKS as this is required for enabling the EBS CSI Driver addon, which is a feature introduced to support EBS-backed volumes on EKS v1.23 and higher. 

More context on this functionality can be found in rancher/eks-operator#170

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md
Comment (martyav) [inline docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md:91]:
```suggestion
These are the minimum set of permissions necessary to access the full functionality of Rancher's EKS driver. You'll need additional permissions for Rancher to provision the `Service Role` and `VPC` resources. If you create these resources **before** you create the cluster, they'll be available when you configure the cluster. 

:::note 
In EKS v1.23 and above, you must use the out-of-tree drivers for EBS backed volumes. You need the EBS CSI Driver add-on permissions to enable the add-on.
:::
```
"EBS CSI Driver add-on" == the specific permission? We should mention the v1.23+ info within the description for the resource in the table.

**Review by martyav — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md
Comment (salasberryfin) [reply docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md:91]: `EBS CSI Driver add-on` refers to the name of the Amazon add-on that must be installed in the EKS cluster to allow interaction with EBS volumes. To install and use this add-on, it is required to add these extra IAM permissions.

**Review by martyav — APPROVED** [review]

**Comment by martyav** [comment]
Reminder that once we get technical approval we'll need to reflect changes to the equivalent page in versioned_docs/version-2.7

**Comment by kkaempf** [comment]
Waiting for https://github.com/rancher/dashboard/issues/9043

**Comment by martyav** [comment]
Added milestone as connected UI issue is slated for 2.8

**Comment by martyav** [comment]
Removed milestone as https://github.com/rancher/dashboard/issues/9043 is slated for 2.8.1 or later

**Comment by martyav** [comment]
Associated issue is now slated for 2.9

**Pull request merged**
